src/xrEngine/device.cpp

```diff
@@ -520,10 +520,10 @@ void CRenderDevice::Run()
 
     // Pre start
     GEnv.Render->MakeContextCurrent(false);
-    deviceReadyToRun.Set();  // ping
+    deviceReadyToRun.Set();
 
-    while (!deviceReadyToRun.Wait(MaximalWaitTime))
```

Comment:
Oh.. So, it pings and receives pong from himself..

Reasonable. I will remember that, to not to repeat such a mistake.